operations/registers.md

```diff
@@ -0,0 +1,97 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Registers
+
+## A register within an outlet.
+
+A register represents a terminal within an outlet, used to issue invoices. Each register is
+uniquely identified within its outlet and can either be physical or virtual.
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and 
+supports the `include` query parameter `outlet` following the [JSON:API specification](../essential-guide/features.md).
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/registers/{id}`
+
+### Response
+
+```javascript
+{
+  "data": {
+    "id": "31b14937-2524-491f-b0a0-dc0a7393ff3f",
+    "type": "registers",
+    "attributes": {
+      "name": "Main Register",
+      "invoices_count": 1200,
+      "index": 3,
+      "virtual": false,
+      "created_at": "2023-10-16T14:30:00Z",
+      "updated_at": "2023-10-19T14:30:00Z"
+    },
+    "relationships": {
+      "outlet": {
+        "data": {
+          "id": "5efa8b3c-b930-4b31-918d-95ab0e212e64",
+          "type": "outlets"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "id": "5efa8b3c-b930-4b31-918d-95ab0e212e64",
+      "type": "outlets",
+      "attributes": {
+        "name": "Downtown Cafe",
+        "address1": "123 Main Street",
+        "address2": "Suite 4B",
+        "city": "New York",
+        "State": "New York",
+        "postal_code": "10001",
+        "index": 5,
+        "created_at": "2023-10-16T13:30:00Z",
+        "updated_at": "2023-10-16T15:30:00Z"
+      }
+    }
+  ]
+}
+```
+
+A related resource object from type registers.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Data` | [register](registers.md#register) | required |  |
+| `Included` | array of [outlet](registers.md#outlet) | optional, max 1 item |  |
+
+#### register
+
+| Property | Type | Contract | Description |
```

Comment:
Thinking... this object definition should be identical for all JSON objects in the API, as it conforms to the JSON:API specification. So we might want to just think extra carefully as to how we describe it in a generic way. What you've got there already is pretty good, maybe we just fill in the descriptions as per the JSON:API spec.